### ISSUE TITLE
Once reachability goes to not available, it wont change again?

### ISSUE THREAD

**Issue by MrNiceMrGood** [RestKit/RestKit#2509]
`setReachabilityStatusChange(_ block: ((AFNetworkReachabilityStatus) -> Swift.Void)!)`
Using this function to track changes in network.
After going to not available , block wont execute again.
I check value of network status and it stays not available even after actually network is available.
Anyone got issue?

**Comment by squatchus** [comment]
yep, experiencing the same issue, it doesn't happen all the time, though.
